rcl_yaml_param_parser/CMakeLists.txt

```diff
@@ -8,6 +8,16 @@ find_package(rcl REQUIRED)
 find_package(libyaml_vendor REQUIRED)
 find_package(yaml REQUIRED)
 
+# Default to C11
+if(NOT CMAKE_C_STANDARD)
+  set(CMAKE_C_STANDARD 11)
```

Comment:
If this package doesnt require C11, let's default to C99 instead as this is the version we target by default: https://github.com/ros2/ros2/wiki/Developer-Guide#c